packages/multichain-network-controller/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "../../tsconfig.packages.json",
+  "compilerOptions": {
+    "baseUrl": "./"
+  },
+  "references": [
+    { "path": "../base-controller" },
+    { "path": "../accounts-controller" },
```

Comment:
Should we remove this for now, since it's not interacting with the accounts controller yet?

Reply:
I would recommend doing so, yes, and adding it in a future PR.